CHANGELOG.md

```diff
@@ -0,0 +1,669 @@
+
+## [v0.8.0-rc2](https://github.com/CosmWasm/cw-plus/tree/v0.8.0-rc2) (2021-08-05)
+
+[Full Changelog](https://github.com/CosmWasm/cw-plus/compare/v0.8.0-rc1...v0.8.0-rc2)
+
+**Closed issues:**
+
+- Upgrade CosmWasm to 0.16.0 [\#377](https://github.com/CosmWasm/cw-plus/issues/377)
+- Upgrade rust to 1.53 [\#372](https://github.com/CosmWasm/cw-plus/issues/372)
+- Implement cw20 logo spec for cw20-base [\#371](https://github.com/CosmWasm/cw-plus/issues/371)
+- multi-test: Pass in API access to BankKeeper [\#353](https://github.com/CosmWasm/cw-plus/issues/353)
+- multi-test: data handling with replies [\#352](https://github.com/CosmWasm/cw-plus/issues/352)
+- multi-test: Add migrate support [\#351](https://github.com/CosmWasm/cw-plus/issues/351)
+- multitest: Ensure Warm sent funds visible to querier [\#347](https://github.com/CosmWasm/cw-plus/issues/347)
+- multitest: Enforce validity checks for returned items [\#341](https://github.com/CosmWasm/cw-plus/issues/341)
+
+**Merged pull requests:**
+
+- Update to Rust 1.53 [\#379](https://github.com/CosmWasm/cw-plus/pull/379) ([ethanfrey](https://github.com/ethanfrey))
+- Upgrade to cosmwasm 0.16 [\#378](https://github.com/CosmWasm/cw-plus/pull/378) ([ethanfrey](https://github.com/ethanfrey))
+- Marketing info for cw20-base contract [\#375](https://github.com/CosmWasm/cw-plus/pull/375) ([hashedone](https://github.com/hashedone))
+- cw20-merkle-airdrop: change hashing to sha256 [\#374](https://github.com/CosmWasm/cw-plus/pull/374) ([orkunkl](https://github.com/orkunkl))
+- Responses validation in multi-test [\#373](https://github.com/CosmWasm/cw-plus/pull/373) ([hashedone](https://github.com/hashedone))
+- Cw20 logo spec [\#370](https://github.com/CosmWasm/cw-plus/pull/370) ([ethanfrey](https://github.com/ethanfrey))
+- Properly handling data in submessages in multi-test [\#369](https://github.com/CosmWasm/cw-plus/pull/369) ([hashedone](https://github.com/hashedone))
+- Abstracting API out of tests internals so it is clearly owned by `App` [\#368](https://github.com/CosmWasm/cw-plus/pull/368) ([hashedone](https://github.com/hashedone))
+- Storage plus doc correction [\#367](https://github.com/CosmWasm/cw-plus/pull/367) ([hashedone](https://github.com/hashedone))
+- Multitest migrate support [\#366](https://github.com/CosmWasm/cw-plus/pull/366) ([ethanfrey](https://github.com/ethanfrey))
+- Reorganizations of contracts in `multi-test::test_utils` [\#365](https://github.com/CosmWasm/cw-plus/pull/365) ([hashedone](https://github.com/hashedone))
+- Implement cw20-merkle-airdrop [\#364](https://github.com/CosmWasm/cw-plus/pull/364) ([orkunkl](https://github.com/orkunkl))
+- Testing sent founds visibility in multi-test [\#363](https://github.com/CosmWasm/cw-plus/pull/363) ([hashedone](https://github.com/hashedone))
+
+## [v0.8.0-rc1](https://github.com/CosmWasm/cw-plus/tree/v0.8.0-rc1) (2021-07-29)
```

Comment:
It would be cool if all the rc's were merge together.

So [0.8.0] shows [0.8.0] + [0.8.0-rc3] + [0.8.0-rc2] + [0.8.0-rc1], as external users are not expected to use the rc's.

Reply:
Unfortunatelly I don't see such option in THIS tool. I could merge those rcs manually.
Later, there's option `BASE` so we could point generator to old changelog and AFAIK it would append only new changes.

Reply:
(btw. I've done just that)